packages/headless/src/modules/manufacturer/repositories/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Manufacturer } from '../../../entity/manufacturer';
```

Comment:
i think `import { Manufacturer } from 'src/entity/manufacturer'` that will be fine